### PR TITLE
Bugfix/lint prettier fix

### PR DESCRIPTION
**Pull request by jackblackCH**
Fixes conflicts between formatting- and linting rules. This is a known side effect while using prettier and eslint out of the box.

Since prettier is here for formatting only and since this is depending only on the line length, some eslint rules like comma dangles and parens rules are in conflict with each other. Basically, it does not make sense to lint them, since it can change immediately by the line length.

That's why the community brought us the "eslint-prettier-plugin", that disables all rules which are taken over by prettier. (Code formatting).

https://github.com/prettier/eslint-config-prettier

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -28,7 +28,21 @@
...
+    "import/no-extraneous-dependencies": [
+      "error",
+      {
+        "devDependencies": [
+          "scripts/**",
+          "rollup.config.js",
+          "**/_demo*.js",
+          "**/story.js",
+          "**/_story*.js",
+          "**/*.story.js",
```
Comment (jackblackCH) [inline .eslintrc]: We have to version for the naming for 'stories'. This is a temp fix that works for all the existing versions. This will be cleaned up in a separate PR.

Comment (LucaMele) [reply .eslintrc]: agree
